src/Test/L0/Listener/CommandSettingsL0.cs

```diff
@@ -35,60 +38,53 @@ public void GetsArg()
             }
         }
 
-        [Fact]
+        [Theory]
         [Trait("Level", "L0")]
         [Trait("Category", nameof(CommandSettings))]
-        public void GetsArgFromEnvVar()
+        [InlineData("agent", "GetAgentName", "some agent")]
```

Comment:
What's the advantage of moving to this approach where these are args? It seems to me like it just makes the tests a little less readable, especially in the `act` stage where we have to pull the method off the class type. Is this in anticipation of adding more cases?

Reply:
That's a fair point. When I did this, I thought there might be more InlineData than there was. I can switch it back to be more explicit since we are only testing one of each type (flag, argument, secret)